data/json/mapgen/stadium_baseball.json

```diff
@@ -0,0 +1,561 @@
+        "│.......|..b_b_b_R,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,R_b_b_b..|E'''''u│",
+        "│.rr.rr.V..b_b_b_R,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,R_b_b_b..|||||ß'a│",
+        "│.rr.rr.V..b_b_b_RRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRR_b_b_b..|eee|''O│",
+        "│.rr.rr.V..b_b____________________________________________________________________b_b..|'''+''O│",
+        "│.......|..b_b_│_bbbbbbbbbb_│_bbbbbbbbbbbbbbbb_│_bbbbbbbbbbbbbbbb_│_bbbbbbbbbbb_│_b_b..|eee|''O│",
+        "│||||||||..b________________________________________________________________________b..||||||+|│",
+        "│..........b___bbbbbbbbbbbb___bbbbbbbbbbbbbbbb___bbbbbbbbbbbbbbbb___bbbbbbbbbbbbb___b..........│",
+        "│..........__________________________________________________________________________..........│",
+        "│|~||||||..│_bbbbbbbbbbbbbb_│_bbbbbbbbbbbbbbbb_│_bbbbbbbbbbbbbbbb_│_bbbbbbbbbbbbbbb_│..||||||~|│",
+        "│...AAA.|......................................&.......................................|.SSS...│",
+        "│...AAA.|..............................................................................|.......│",
+        "│&......|||||||||||...|||||||||||.....|..................|.....|||||||||||...|||||||||||m....zz│",
+        "│......&|''SSS'''a|...|a'''SSS''|.....|..............2...|.....|''SSS'''a|...|a'''SSS''|m....zz│",
+        "│&......|'''''''''|...|'''''''''|.<.<.|..1...............|.<.<.|'''''''''|...|'''''''''|m.....&│",
+        "│&......|+|+|+|+|'|...|'|+|+|+|+|.|.|.|..................|.|.|.|+|+|+|+|'|...|'|+|+|+|+|m......│",
+        "│.......|'|'|'|'|'|...|'|'|'|'|'|.....|..................|.....|'|'|'|'|'|...|'|'|'|'|'|..&...A│",
+        "│6......|T|T|T|T|''...''|T|T|T|T|.....|..................|.....|T|T|T|T|''...''|T|T|T|T|....&&A│",
+        "││#####│││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││││"
+      ],
+      "palettes": [ "stadium_palette" ],
+      "terrain": { "#": "t_door_metal_locked", "6": "t_gates_mech_control_lab" },
+      "items": {
+        "E": { "item": "softdrinks_canned", "chance": 40, "repeat": [ 1, 20 ] },
+        "r": { "item": "baseball_souvenir", "chance": 50 },
+        "e": { "item": "fast_fridge", "chance": 25, "repeat": [ 1, 20 ] },
+        "u": { "item": "fast_kitchen", "chance": 60, "repeat": [ 1, 10 ] },
+        "é": { "item": "hotdog_fridge", "chance": 70 }
+      },
+      "place_item": [ { "item": "napkin", "x": 93, "y": 75, "chance": 70 }, { "item": "napkin", "x": 5, "y": 48, "chance": 70 } ],
+      "place_items": [
+        { "item": "ketchup_sealed_rng", "x": 5, "y": 51, "chance": 60, "repeat": [ 1, 2 ] },
+        { "item": "mustard_sealed_rng", "x": 5, "y": 51, "chance": 60, "repeat": [ 1, 2 ] }
+      ],
+      "place_monsters": [
+        { "monster": "GROUP_HOUSE", "x": [ 0, 23 ], "y": [ 0, 23 ], "chance": 5 },
```

Comment:
Strongly advise against using `place_monsters` flag, since it spawns ALL of the monsters in the group.

Reply:
I'll address this in my next PR.